lib/screens/stops/stop.ex

```diff
@@ -15,6 +15,196 @@ defmodule Screens.Stops.Stop do
+    {"place-brico", {"Packards Corner", "Packards Cn"}},
+    {"place-harvd", {"Harvard Avenue", "Harvard Ave"}},
+    {"place-grigg", {"Griggs Street", "Griggs St"}},
+    {"place-alsgr", {"Allston Street", "Allston St"}},
+    {"place-wrnst", {"Warren Street", "Warren St"}},
+    {"place-wascm", {"Washington Street", "Washington"}},
+    {"place-sthld", {"Sutherland Road", "Sutherland"}},
+    {"place-chswk", {"Chiswick Road", "Chiswick Rd"}},
+    {"place-chill", {"Chestnut Hill Avenue", "Chestnut Hl"}},
+    {"place-sougr", {"South Street", "South St"}},
+    {"place-lake", {"Boston College", "Boston Coll"}}
+  ]
+
+  @green_line_c_stops [
+    {"place-gover", {"Government Center", "Gov't Ctr"}},
+    {"place-pktrm", {"Park Street", "Park St"}},
+    {"place-boyls", {"Boylston", "Boylston"}},
+    {"place-armnl", {"Arlington", "Arlington"}},
+    {"place-coecl", {"Copley", "Copley"}},
+    {"place-hymnl", {"Hynes Convention Center", "Hynes"}},
+    {"place-kencl", {"Kenmore", "Kenmore"}},
+    {"place-smary", {"Saint Mary's Street", "St. Mary's"}},
+    {"place-hwsst", {"Hawes Street", "Hawes St"}},
+    {"place-kntst", {"Kent Street", "Kent St"}},
+    {"place-stpul", {"Saint Paul Street", "St. Paul St"}},
+    {"place-cool", {"Coolidge Corner", "Coolidge Cn"}},
+    {"place-sumav", {"Summit Avenue", "Summit Ave"}},
+    {"place-bndhl", {"Brandon Hall", "Brandon Hll"}},
+    {"place-fbkst", {"Fairbanks Street", "Fairbanks"}},
+    {"place-bcnwa", {"Wash Sq", "Washington"}},
+    {"place-tapst", {"Tappan Street", "Tappan St"}},
+    {"place-denrd", {"Dean Road", "Dean Rd"}},
+    {"place-engav", {"Englewood Avenue", "Englew'd Av"}},
+    {"place-clmnl", {"Clvlnd Circ", "Clvlnd Cir"}}
+  ]
+
+  @green_line_d_stops [
+    {"place-north", {"North Station", "North Sta"}},
+    {"place-haecl", {"Haymarket", "Haymarket"}},
+    {"place-gover", {"Government Center", "Gov't Ctr"}},
+    {"place-pktrm", {"Park Street", "Park St"}},
+    {"place-boyls", {"Boylston", "Boylston"}},
+    {"place-armnl", {"Arlington", "Arlington"}},
+    {"place-coecl", {"Copley", "Copley"}},
+    {"place-hymnl", {"Hynes Convention Center", "Hynes"}},
+    {"place-kencl", {"Kenmore", "Kenmore"}},
+    {"place-fenwy", {"Fenway", "Fenway"}},
+    {"place-longw", {"Longwood", "Longwood"}},
+    {"place-bvmnl", {"Brookline Village", "B'kline Vil"}},
+    {"place-brkhl", {"Brookline Hills", "B'kline Hls"}},
+    {"place-bcnfd", {"Beaconsfield", "B'consfield"}},
+    {"place-rsmnl", {"Reservoir", "Reservoir"}},
+    {"place-chhil", {"Chestnut Hill", "Chestnut Hl"}},
+    {"place-newto", {"Newton Centre", "Newton Ctr"}},
+    {"place-newtn", {"Newton Highlands", "Newton Hlnd"}},
+    {"place-eliot", {"Eliot", "Eliot"}},
+    {"place-waban", {"Waban", "Waban"}},
+    {"place-woodl", {"Woodland", "Woodland"}},
+    {"place-river", {"Riverside", "Riverside"}}
+  ]
+
+  @green_line_e_stops [
+    {"place-north", {"North Station", "North Sta"}},
```

Comment:
So that's a good question for more than just this widget. This is code that originally was only being used by `SubwayStatus`. We might need to make a GLX related task for our apps/widgets.

Reply:
https://app.asana.com/0/1185117109217422/1202042701703726/f